app/controllers/album.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const errors = require('../errors'),
+  { getAll } = require('../services/album'),
+  logger = require('../logger');
+
+exports.albumList = async (req, res, next) => {
+  try {
+    const albums = await getAll();
```

Comment:
Await seems not necessary here, one simple promise with `.then` and `.catch` should be fine